File: app/src/main/java/com/gratus/workoutrepo/model/DayWorkout.java
package com.gratus.workoutrepo.model; //class moved

import androidx.annotation.Keep;

@Keep // Add this annotation
public class DayWorkout {
    public String dayName; // "MONDAY", etc.
    public String workoutType;
    public String majorWorkouts;
    public String minorWorkouts;
    public String notes;

    public DayWorkout(String dayName) {
        this.dayName = dayName;
        this.workoutType = "";
        this.majorWorkouts = "";
        this.minorWorkouts = "";
        this.notes = "";
    }
}

File: app/src/main/java/com/gratus/workoutrepo/routine/model/DayWorkout.kt
package com.gratus.workoutrepo.routine.model; //class moved

import androidx.annotation.Keep;

@Keep // Add this annotation
public class DayWorkout {
    public String dayName; // "MONDAY", etc.
    public String workoutType;
    public String majorWorkouts;
    public String minorWorkouts;
    public String majorLabel;
    public String minorLabel;
    public String notes;

    public DayWorkout(String dayName) {
        this.dayName = dayName;
        this.workoutType = "";
        this.majorWorkouts = "";
        this.minorWorkouts = "";
        this.majorLabel = "";
        this.minorLabel = "";
        this.notes = "";
    }
}